app/services/active_storage/service/sdr_service.rb

```diff
@@ -15,6 +15,11 @@ def download(key, &block)
         end
       end
 
+      def delete(key)
+        Honeybadger.context(key)
+        Honeybadger.notify('SdrService#delete called unexpectedly - possible attempt to purge work in preservation')
+      end
```

Comment:
Since we're manually notifying HB here, we can include the context in the `.notify` call:

```ruby
Honeybadger.notify('...', { key: })
```